back/src/server.ts

```diff
@@ -54,16 +54,22 @@ app.use(function (
 });
 
 // Socket.io initialization
-const io = new Server(httpServer, {});
+const io = new Server(httpServer, {
+  cors: {
+    origin: 'http://localhost:8100',
+  },
+});
 
 io.on('connection', (socket) => {
   const userId = socket.handshake.auth.userId;
 
   if (!userId) {
-    console.log('No userId for this user ! ', socket);
+    console.log('No userId for this user !');
```

Comment:
```suggestion

```

Reply:
IMO it's better to keep it, it ensures you'll see why the back is panicking when looking at the logs.